### ISSUE TITLE
Found an embedded parameter "${AWS::Partition}" outside of an "Fn::Sub" at Resources

### ISSUE THREAD

**Issue by virtlima** [aws-cloudformation/cfn-python-lint#1645]
*cfn-lint version: (`cfn-lint --version`)*
cfn-lint 0.29.5

*Description of issue.*
I have a CloudFormation Template embedded into a AWS Systems Manager Automation Document. 

Please provide as much information as possible:
* Template linting issues:

`E1029 Found an embedded parameter "${AWS::Partition}" outside of an "Fn::Sub" at Resources/DNSForwarderSetup/Properties/Content/mainSteps/0/inputs/TemplateBody
ad-3.template:433:13`

This is actually okay, I am using !Sub correctly in an Embedded CloudFormation.... 

  * Please provide a CloudFormation sample that generated the issue.

https://github.com/aws-quickstart/quickstart-microsoft-activedirectory/blob/b071c3def8d9a30fd055571c9603cad10a753f40/templates/ad-3.template#L433

  * If present, please add links to the (official) documentation for clarification.
  * Validate if the issue still exists with the latest version of `cfn-lint` and/or the latest Spec files
* Feature request:
  * Please provide argumentation about the missing feature. Context is key!
This errors if I use !Sub or Fn::Sub: when it actually correct.. 

Cfn-lint uses the [CloudFormation Resource Specifications](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-resource-specification.html) as the base to do validation. These files are included as part of the application version. Please update to the latest version of `cfn-lint` or update the spec files manually (`cfn-lint -u`)

**Comment by PatMyron** [comment]
Thanks for the report. Should be fixed for the next upcoming release

Can workaround with [resource-level ignores](https://github.com/aws-cloudformation/cfn-python-lint#resource-based-metadata) until then:

```yaml
  Resource:
    Type: AWS::SSM::Document
    Metadata:
      cfn-lint:
        config:
          ignore_checks:
          - E1029
```
